server/src/file/file.entity.ts

```diff
@@ -0,0 +1,18 @@
+import { Application } from '@/application/application.entity';
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { CustomBaseEntity } from '../common/custom-base.entity';
+
+@Entity()
+export class File extends CustomBaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column({ type: 'varchar', length: 100, nullable: false })
+  name: string;
+
+  @Column({ type: 'bytea', nullable: false })
```

Comment:
Just double checking we're intending to store files directly in the database rather than in something like S3?

Reply:
Karthikk said to implement it like this for now, so I guess it's to speed thing us/for convenience for now. If there's a better way - let me know!

Reply:
I'll confirm in some time!